enterprise/app/code/code_empty.tsx

```diff
@@ -4,13 +4,15 @@ import { invocation } from "../../../proto/invocation_ts_proto";
 import format from "../../../app/format/format";
 
 interface State {
-  repoStats: invocation.InvocationStat[];
+  reposWithStats: string[];
+  linkedRepos: string[];
   loading: boolean;
 }
 
 export default class CodeEmptyStateComponent extends React.Component {
```

Comment:
```suggestion
export default class CodeEmptyStateComponent extends React.Component<{}, State> {
```

(this will probably surface some type errors :stuck_out_tongue: )

Reply:
Good catch, fixed